docs/manual-EN/2.query-language/0.README.md

```diff
@@ -1,3 +1,66 @@
 # Reader
 
 This chapter is for those who want to use `Nebula Graph` query language.
+
+## Placeholder Identifiers and Values
+
+The query language of **Nebula Graph**, i.e. the nGQL, refers to the following standards in design:
```

Comment:
The query language of **Nebula Graph** is the nGQL. Refer to the following standards of nGQL: